.github/workflows/ci.yml

```diff
@@ -236,8 +237,8 @@ jobs:
 
     - name: Install packages
       # TODO: check which fonts are really needed
-      run:  zypper --non-interactive install --no-recommends gcc gcc-c++ make openssl-devel ruby-devel
-            npm augeas-devel cockpit dbus-1 xorg-x11-server libgtk-3-0
+      run:  zypper --non-interactive install --no-recommends gcc-c++ ruby-devel
+            'npm>=18' augeas-devel cockpit xorg-x11-server libgtk-3-0
```

Comment:
Ideally we should somehow automate this or share the list of packages. For the regular YaST packages you can run `rake build_dependencies:list` to get the build dependencies.

Unfortunately that does not work for the D-Installer correctly because it has multiple `*.spec` files in different subdirectories. I'll check if could improve that...